### PR TITLE
HTBHF-1801 Added change links and hint text to send-code page

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

File: src/test/session-details-provider/README.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ without those tests needing to actually receive the code via text or email.
 
 `session-details-manifest.yml` is used to deploy it to cloud foundry, by running this command from the root of the project:
 ```
-cf push -f src/test/session-details-provider/session-details-manifest.yml --var session_details_app_name=htbhf-session-details-staging --var session_secret=ReplaceWithTheRealSessionSecret
+cf push -f src/test/session-details-provider/session-details-manifest.yml \
```
Comment (YetAnotherMatt) [inline src/test/session-details-provider/README.md]: Unrelated change that I forgot was in my workspace. I added the missing `session_details_host_name` variable to the example push command. Let me know if it needs to come out...

File: src/test/acceptance/features/send-code.feature [hunk 2]
```diff
@@ -8,6 +8,8 @@ Feature: We're going to send you a code
 
   Scenario: Text and Email are displayed
     Then Text and Email options are displayed on the send code page
+    And The change-text link points to the phone number page
```
Comment (mikeSmith08) [inline src/test/acceptance/features/send-code.feature]: Why the need for dashes in `change-text` and `change-email`?

File: src/test/common/steps/send-code-steps.js [hunk 3]
```diff
@@ -23,13 +23,33 @@ Then(/^Text and Email options are displayed on the send code page$/, async funct
...
 
 Then(/^I am informed that I need to select an option for send code$/, async function () {
   await assertErrorHeaderTextPresent(pages.sendCode)
   await assertSendCodePresent()
 })
 
+Then(/^The change-text link points to the phone number page$/, async function () {
+  const changeLink = await pages.sendCode.getChangeTextLink()
+  const linkHref = await changeLink.getAttribute('href')
+  expect(linkHref).to.be.equal(`${pages.url}/phone-number`)
```
Comment (mikeSmith08) [inline src/test/common/steps/send-code-steps.js]: Can we get the link from a page object please? Same with below too.

File: src/web/server/locales/en/common.json [hunk 4]
```diff
@@ -90,7 +90,8 @@
     "title": "We’re sending you a code",
     "heading": "We’re sending you a code",
     "hint": "Select how you would like to receive it",
-    "explanation": "We need this to confirm your identity and to prevent fraud."
+    "explanation": "We need this to confirm your identity and to prevent fraud.",
+    "change": "change"
```
Comment (mikeSmith08) [inline src/web/server/locales/en/common.json]: Any reason its `change` rather than `Change`?

File: src/web/views/send-code.njk [hunk 5]
```diff
@@ -19,14 +19,17 @@
                 value: "text",
                 text: text,
                 checked : claim.channelForCode === "text",
-                conditional: {
-                html: eddHtml
-            }
+                hint: {
+                    html: claim.phoneNumber + " <a href=\"/phone-number\" id=\"change-text\">" + change + "</a>"
```
Comment (mikeSmith08) [inline src/web/views/send-code.njk]: Shouldn't this use the value from the translations file? Same below too?

File: src/test/acceptance/features/send-code.feature (hunk 2 above)
Comment (YetAnotherMatt) [reply src/test/acceptance/features/send-code.feature]: No reason - that was the id I used. I'll remove it from the step name.

File: src/web/server/locales/en/common.json (hunk 4 above)
Comment (YetAnotherMatt) [reply src/web/server/locales/en/common.json]: Because it looks better :)
Seriously, that was the UX preference.

File: src/web/views/send-code.njk (hunk 5 above)
Comment (YetAnotherMatt) [reply src/web/views/send-code.njk]: it is!

File: src/test/common/steps/send-code-steps.js (hunk 3 above)
Comment (YetAnotherMatt) [reply src/test/common/steps/send-code-steps.js]: done

File: src/web/server/locales/en/common.json (hunk 4 above)
Comment (mikeSmith08) [reply src/web/server/locales/en/common.json]: That's fine, just thought I'd check.

File: src/web/views/send-code.njk (hunk 5 above)
Comment (mikeSmith08) [reply src/web/views/send-code.njk]: Oh, yes, missed that.
